.travis/environment.yml

```diff
@@ -0,0 +1,140 @@
+name: sample-factory
```

Comment:
Why do we have two versions of environment.yml?
If they are different, can you put the difference in a patch file (like the output git diff generates), and patch the main file in this script? This way it will be a lot more apparent.
Otherwise, code duplication is really bad, especially of this sort. We will change something in main .yml and forget to change it here.

Reply:
The problem is with Travis. Travis would require CPU only versions of torch

Reply:
Patch file created